doc/architecture/decisions/0011-control-applicant-access-using-delius-team-caseload.md

```diff
@@ -0,0 +1,168 @@
+# 11. Control "applicant" access using Delius team caseload
+
+Date: 2023-01-23
+
+## Status
+
+Accepted
+
+## Context
+
+The service will manage all but one of its roles internally. This ADR concerns that one
+exception: the "applicant" role which is not modelled explicitly within the service, but
+is derived from caseload information from Delius.
+
+The Approved Premises service exposes personal data from NOMIS and from OASys and so
+it's necessary to restrict creating and viewing "applications" to probation officers with an
+appropriate caseload.
+
+
+
+### Overview of the service's roles
+
+#### Applicant
+**Role**: `APPLICANT`  
+**Description**: probation officer making application  
+**Authorised to**:
+- create an application subject to the restrictions on their Delius user account, with 
+  which they've logged in to the service
+- withdraw an application at any stage of the process if there are material changes in 
+  circumstances. These are to be defined but would include death.
+- view applications which they and team mates have made, including assessment and matching notes
+  which are added during the downstream processes
+- respond to requests for information raised during the assessment process (not MVP)
+
+#### Workflow manager
+**Role**: `WORKFLOW_MANAGER`   
+**Description**: CRU staff ensuring that assessment and allocation tasks are completed  
+**Authorised to**:
+- view all applications and their allocations to staff members
+- reassign applications to different staff members as necessary
+- mark application as withdrawn (e.g. if person has died)
+- ask applicant for further info, independent of the assessment itself (in MVP this is 
+  limited to adding notes based on off-line communication)
+
+#### Assessor
+**Role**: `ASSESSOR`  
+**Description**: Staff member who is tasked with assessing specific applications  
+**Authorised to**:
+- assess a particular application
+- add additional assessment content (recommendations, clarifications etc) to an application
```

Comment:
Assessors _will_ be able to ask an applicant for further info too

Reply:
removed all this details from the ADR